### ISSUE TITLE
Suggestion: sensor.circadian_values attributes

### ISSUE THREAD

**Issue by Agratam** [claytonjn/hass-circadian_lighting#86]
Hi, i started implementing this integration yesterday and Ii bumped into a couple of issues for which I still struggle to find a workaround.
I needed to pass the Circardian levels to Nodered and started using _sensor.circadian_values _attributes__

Then I realized i needed the Brightness too anc greates a custom sensor:

`- platform: template
  sensors:
    circadian_values_custom:
      friendly_name: "Light Auto Setting"
      value_template: "{{ states.sensor.circadian_values.state }}"
      attribute_templates:
        brightness: "{{ state_attr('switch.circadian_lighting_colore_camera', 'brightness') | int }}"
        kelvin: "{{ state_attr('sensor.circadian_values', 'colortemp') | int }}"
        rgb_color: "{{ state_attr('sensor.circadian_values', 'rgb_color')}}"
        xy_color: "{{ state_attr('sensor.circadian_values', 'xy_col`

And then realized that the _rgb_color_ attributes is not a string of integer.
I'm sure there is a way to fix this on my side, but I'm not that skilled, and need some try and error...

Long story short It would be useful to have rgb_color as string of integer and brightness included in the sensor attrinutes.

Thanks

Matteo

**Comment by claytonjn** [comment]
Brightness cannot be included in the sensor because it varies depending on the switch configuration. As far as rgb color is concerned, technically it's a tuple object - do you really want it to be a string instead, and is there a particular reason that you need the values to be integers?

I'm a bit confused why you'd create the template sensor just to add brightness, when you could just access brightness from the switch in Nodered same as accessing it from the template sensor you created. What are you ultimately trying to do with the values in nodered?

**Comment by Agratam** [comment]
Maybe I'm not using the plugin in the "ortodox" way. I see all cicardian switches set on the same Brightness.

_  is there a particular reason that you need the values to be integers?_
HA service light.turn_on requires a a list containing three integers between 0 and 255 representing the RGB. I'm not able to use the rgb data as is to turn on a light.

_- What are you ultimately trying to do with the values in nodered?_
I'm using a global variable (one per light typology) to store the "current light setting" to be used when turning on a light.
I also have scenes defined in global variable, When I change a scene it's data replaces the "current light setting"
I defined an "Auto" scene which is using circadian values to turn on the light and i was using sensor.circadian_values to keep Auto scene data updated.
When I select a scene different than auto, the cicardian switchwes are turned off and I'm in a sort of manual mode for all the lights.

Matteo

**Comment by LukeDefeo** [comment]
You can parse the string into an int, the developer cant change the type of the sensor without breaking people
